src/lib/utilities/pre-loader/pre-loader.scss

```diff
@@ -16,7 +16,7 @@
   z-index: 10000;
 
   /* light-theme */
-  background-color: map-get($mat-light-theme-background, background);
+  background-color: $white-87-opacity;
```

Comment:
These should now use the dedicated variables for page background rather than manually defining a color. 

For example, here it would be:
`background-color: mat-color($background, background);`

You don't even have to have a separate definition for the dark theme version, it will just swap to its dark theme value automatically.

https://github.com/Teradata/vantage-ui-platform/blob/439c784d32cb4841e9e9d3ea40dc3ab48ac7f9f0/src/lib/theme/_palette.scss#L154

Reply:
`mat-color($background, background);` does not work as `$background` is not defined. How can we define this variable in a way it automatically takes the theme into account?

Reply:
Every time you create a component theme file, you have to define the foreground and background maps again in that file to get access to the color map within that file. So you'd just have to do this at the top of the file to get the background color map:

`$background: map-get($theme, background);`

Check out the top of the _overrides file to see it in action.

If it's not swapping based on theme properly, you'll probably need to wrap the whole theme file in a mixin. _overrides does this, here's more on that: https://material.angular.io/guide/theming-your-components

Reply:
This file, pre-loader.scss, is a special file since it is separate context from the theme.scss where we do the theme initialization, and I do not want to be pulling more css than I need to.